build.gradle

```diff
@@ -21,28 +23,29 @@ targetCompatibility = JavaVersion.VERSION_1_6
 
 
 repositories {
+    google()
+    mavenCentral()
     jcenter()
 }
 
 
 dependencies {
     // Kotlin
-    compile "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
-    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+    implementation("org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion")
```

Comment:
nit: would be nice to have the same pattern here as the one below this too (either parenthesis)